### ISSUE TITLE
MyGet feed returns 404 (https://www.myget.org/F/imagesharp/api/v3/index.json)

### ISSUE THREAD

**Issue by Nikoshi** [eurofurence/ef-app_backend-dotnet-core#13]
### Steps to reproduce:
- execute `dotnet build` in `src/Eurofurence.App.Server.KestelHost`

### Result:
```
/opt/dotnet/sdk/2.1.500/NuGet.targets(114,5): error : Unable to load the service index for source https://www.myget.org/F/imagesharp/api/v3/index.json. [/home/icetiger/dev/eurofurence/ef-app_backend-dotnet-core/src/Eurofurence.App.Server.KestrelHost/Eurofurence.App.Server.KestrelHost.csproj]
/opt/dotnet/sdk/2.1.500/NuGet.targets(114,5): error :   Response status code does not indicate success: 404 (Feed does not exist.). [/home/icetiger/dev/eurofurence/ef-app_backend-dotnet-core/src/Eurofurence.App.Server.KestrelHost/Eurofurence.App.Server.KestrelHost.csproj]
```

### Expected result:
dotnet should build the api

**Comment by Pinselohrkater** [comment]
Thanks for pointing this out!

When I first introduced ImageSharp, it was still in early alpha and only available on MyGet - I've updated the references and some lines to switch over to the NuGet beta package instead. I guess you're the first to notice because this usually gets build on our server or on my machine - and in both places the referenced packages are still in the NuGet package cache.

I've yet to do some smoke-tests on the areas affected, but it should restore & build for you.
